File: app/src/main/java/com/lzy/testproject/MainActivity.java
package com.lzy.testproject;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.widget.TextView;
import android.widget.Toast;

import com.lzy.annotation.BindView;
import com.lzy.annotation.Losd;

import java.util.ArrayList;
import java.util.List;

public class MainActivity extends AppCompatActivity {
	@BindView(R.id.tv)
	TextView textView;

	@BindView(R.id.tv1)
	TextView textView1;

	@Override
	protected void onCreate(Bundle savedInstanceState) {
		super.onCreate(savedInstanceState);
		setContentView(R.layout.activity_main);


	}

	private void test(){

	}
}

File: app/src/main/java/com/lzy/testproject/kotlin/MainActivity.kt
package com.lzy.testproject.kotlin

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.util.Log
import com.lzy.testproject.R

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main2)

        val name = if (savedInstanceState != null) "sd" else "Kotlin"
        Log.d("lzy","Hello $name")
    }
}
